runtime/src/main/java/org/corfudb/runtime/view/LayoutView.java

```diff
@@ -161,9 +167,22 @@ public Layout prepare(long epoch, long rank)
         // Return any layouts that have been proposed before.
         List<LayoutPrepareResponse> list = Arrays.stream(acceptList)
```

Comment:
What happens if the quorum condition is met and we did not get the latest proposed layout in that ? Let's say we had 3 nodes, 2 nodes responded but the responses only had layout 1 and not the latest proposed layout 2.
We should decide that if we want to short circuit as soon as the quorum is reached ?

Reply:
Isn't that acceptable as it means that the propose wasn't successful in acquiring a majority of votes. Thus we can safely assume null layout ?

Reply:
Sorry I did not mean anything is wrong here. The behavior is acceptable (I just want to cover the loose ends even though they are not part of correctness)
I am not sure what you mean by null but Here is what one acceptable behaviour
N1,    N2,      N3
L1,    null,      L2        quorum (N1, N2) then L1 , quorum (N1, N3) then L2, quorum (N2, N3) then L2  
L1,   null,      null        quorum(N1, N2) then L1, quorum (N1, N3) then L1,  quorum (N2, N3) then null

The intended question was why would you not want to wait for all three responses (up to timeout) because if all three responses are back then in the second case
quorum(N1, N2, N3)  then (as per the code) L1 (both L1 and null are acceptable).
Also we want to test these cases.